org.scala-ide.sdt.core/src/org/scalaide/core/internal/extensions/autoedits/CloseAngleBracketCreator.scala

```diff
@@ -0,0 +1,13 @@
+package org.scalaide.core.internal.extensions.autoedits
+
+import org.scalaide.core.text.Document
+import org.scalaide.core.text.TextChange
+import org.scalaide.extensions.autoedits.CloseAngleBracket
+
+object CloseAngleBracketCreator {
```

Comment:
I don't like `CloseAngleBracketCreator.create`, maybe `CloseAngleBracket.create` to remove duplication?